tests/Domain/DateOfIssueTest.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+namespace DDDWorkshop\Domain;
+
+use DateTimeImmutable;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @coversDefaultClass \DDDWorkshop\Domain\DateOfIssue
+ */
+class DateOfIssueTest extends TestCase
+{
+    /**
+     * @throws \Exception
+     */
+    public function testEquals(): void
```

Comment:
I suggest splitting this test to two - `testSuccessEquals` and `testFailEquals` to improve readability